### PR TITLE
Add likelihood indicator guideline

### PR DESCRIPTION
**Pull request by gene1wood**

### CODE REVIEW

File: docs/guidelines/likelihood_indicators.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+| LOW     | MEDIUM  |
+| MEDIUM  | HIGH    |
+| HIGH    | MAXIMUM |
+| MAXIMUM | MAXIMUM |
+</div>
+
+# Communicating the likelihood indicator
+
+Likelihood indicators should be sent in the 
+[standard ServiceUI format](https://mana.mozilla.org/wiki/display/SECURITY/Risk+Integration+Architecture#RiskIntegrationArchitecture-RESTInterfaces(ofServiceUI)
```
[the remark this answers is not in the document]
Comment (jeffbryner) [reply docs/guidelines/likelihood_indicators.md]: Yes, should be removed for now as it's wildly out of sync, even to what's in the RRA.

File: docs/guidelines/likelihood_indicators.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+Service risk is composed both of the impact when a risk is manifested as well as
+the likelihood that the risk will manifest. Impact can be assessed in a
+[Rapid Risk Assessment](https://wiki.mozilla.org/Security/Risk_management/Rapid_Risk_Assessment#Recording_risk_impacts)
+and is primarily based on the data which the service handles.
+Likelihood on the other hand is primarily driven by the presence or absence of
+security controls in the service.
+
+What follows is a methodology for associating likelihood indicators with
+security controls. These likelihood indicators can then be integrated into the
+[Risk Integration Architecture](https://mana.mozilla.org/wiki/display/SECURITY/Risk+Integration+Architecture).
```
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: How about we link to the page here and change it down the road if we have something better to put there?

File: docs/guidelines/likelihood_indicators.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: Oh. Is there an alternative method that we should put in place?

File: docs/guidelines/likelihood_indicators.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+When determining the likelihood indicator for a security control consider
+
+* How easy is it for a threat agent to determine that the control is missing
+* How easy is it for a threat agent to exploit the fact that the control is
+  missing
+* How well known are the paths of exploitation mad possible by the absence of
+  this security control
+* Are there current ongoing attacks on other services which are protected by
+  this security control
```
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: Good idea, added.

File: docs/guidelines/likelihood_indicators.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
+---
```
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: Once we hash out the content, let's merge and then I'll make a new PR that moves this into that location?

File: docs/guidelines/likelihood_indicators.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  in MAXIMUM risk.
+
+# Determining the likelihood indicator for a security control
+
+When determining the likelihood indicator for a security control consider
+
+* How easy is it for a threat agent to determine that the control is missing
+* How easy is it for a threat agent to exploit the fact that the control is
+  missing
+* How well known are the paths of exploitation mad possible by the absence of
```
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: Good idea, I've changed this to

> To what degree do missing security controls make it easier to discover vulnerabilities in the service

**Comment by gene1wood** [comment]
@gdestuynder Thank you for the review. I've merged your suggestions, added changes based on your comments and replied to the remainder of the conversations. Would you take a look over the remaining conversations, resolve them if it sounds good, or reply?

File: docs/guidelines/likelihood_indicators.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: I've opened #91 to do this.

File: docs/guidelines/likelihood_indicators.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gdestuynder) [reply docs/guidelines/likelihood_indicators.md]: I think I'd just drop it until we have a place that receive them again, as per the comment with jeff on it we're not actually using them right now. We should, but we aren't ;-)

**Comment by gdestuynder** [comment]
I think the only remaining thing is to figure out where to send these, or to drop the URL and reference to service API until we have a place to actually send them. Else, r+!

File: docs/guidelines/likelihood_indicators.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: Ok, I'll just remove it.

File: docs/guidelines/likelihood_indicators.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gene1wood) [reply docs/guidelines/likelihood_indicators.md]: Ok, done.

**Comment by gene1wood** [comment]
Cool, I've dropped references to it.

@gdestuynder would you render your r+ into a review approval (merging is blocked)?

**Pull request merged**